ngraph/test/visitors/op/convolution.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "ngraph/op/util/attr_types.hpp"
+#include "ngraph/opsets/opset1.hpp"
+#include "ngraph/opsets/opset3.hpp"
+#include "ngraph/opsets/opset4.hpp"
+#include "ngraph/opsets/opset5.hpp"
+#include "util/visitor.hpp"
+
+using namespace std;
+using namespace ngraph;
+using ngraph::test::NodeBuilder;
+using ngraph::test::ValueMap;
+
+TEST(attributes, convolution) {
+    NodeBuilder::get_ops().register_factory<op::v1::Convolution>();
+    auto data = make_shared<op::Parameter>(element::f32, Shape{1, 16, 124, 124});
+    auto filters = make_shared<op::Parameter>(element::f32, Shape{2, 16, 3, 3});
+    auto strides = Strides{1, 1};
+    auto pads_begin = CoordinateDiff{1, 2};
+    auto pads_end = CoordinateDiff{1, 2};
+    auto dilations = Strides{1, 1};
+    auto convolution = make_shared<op::v1::Convolution>(data,
+                                                                    filters,
```

Comment:
I think this should be algned

Reply:
Applied it and passed the clang-format! Thanks!